src/SFA.DAS.PR.Jobs.MessageHandlers.TestHarness/RaiseEventService.cs

```diff
@@ -1,7 +1,7 @@
-﻿using System.Diagnostics.CodeAnalysis;
-using Microsoft.Extensions.Hosting;
+﻿using Microsoft.Extensions.Hosting;
 using SFA.DAS.EmployerAccounts.Messages.Events;
 using SFA.DAS.PR.Jobs.Functions;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Your VS settings are incorrect, the Systems should be on the top. See Wow page for correct settings.